Marlin/src/MarlinCore.cpp

```diff
@@ -487,7 +493,7 @@ inline void manage_inactivity(const bool no_stepper_sleep=false) {
     // Exceeded threshold and we can confirm that it was not accidental
     // KILL the machine
     // ----------------------------------------------------------------
-    if (killCount >= KILL_DELAY) {
+    if (killCount >= killDelay) {
```

Comment:
Is `killDelay` needed as a variable for any reason, or could you just switch this back to `KILL_DELAY` and eliminate the variable?

Reply:
My thought was to leave a safety net in case some systems are far noisier than we expect. This leaves a simple path should support issues come in to add a define in the configs and find where the limit is on the worst systems.

After some time, I expect it could be however I feel the caution is prudent ATM.

Reply:
What I meant is that you could just use the constant directly here, and completely eliminate `killDelay`. The local doesn't seem to provide any value.

```suggestion
    if (killCount >= KILL_DELAY) {
```

Reply:
Okay, now I understand what you mean! Yeah I've got no problem with that, just left it as was originally. I'll push that change when I'm back in front of my laptop later.